.gitignore

```diff
@@ -14,3 +14,6 @@ docs/build/
 
 # Auto-generated by setup.py
 traits/_version.py
+
+# Cython C files
+traits/ctraits.c
```

Comment:
We may want to rethink this at some point: when we package things up for distribution, we're going to want to include the generated C source in the source distribution (so that we're in control of which Cython version is getting used, and we don't need to support users who've used some random later version of Cython when installing). That doesn't necessarily mean that we need the C source in the Git repo, though.

I admit it's not clear to me how distribution would work right now; I haven't though through the details. Time to open another Cython-labelled issue?